### PR TITLE
ARM network application gateways implemented

### PR DESCRIPTION
**Pull request by MikhailTryakhov**

### CODE REVIEW

**Comment by romangromov** [comment]
@MikhailTryakhov Note: test cases are missed from this PR

File: lib/commands/arm/network/network._js [hunk 1]
```diff
@@ -2361,6 +2362,537 @@ exports.init = function (cli) {
...
+        }
+      }
+
+      var networkManagementClient = getNetworkManagementClient(options);
+      var appGateway = new AppGateway(cli, networkManagementClient);
+      appGateway.addFrontendIp(resourceGroup, gatewayName, name, options, _);
+    });
+
+  appGatewayFrontendIp.command('remove [resource-group] [gateway-name] [name]')
+    .description($('Remove a frontend ip configuration from an application gateway'))
```
Comment (amarzavery) [inline lib/commands/arm/network/network._js]: AFAIK, we only use the verb delete. Please change remove to delete everywhere in this PR.

**Comment by amarzavery** [comment]
@MikhailTryakhov - What is the plan for tests ?

**Comment by MikhailTryakhov** [comment]
Hi @amarzavery 
thank you for review. Could you please clarify, is test plan you mean - the file attached arm.network.application-gateway-tests.js?

**Comment by amarzavery** [comment]
I can see the test file now. But where are the test recordings ?

**Comment by MikhailTryakhov** [comment]
@amarzavery I'm still recording them.
App gateway commands are long-runnung (20 mins for most of commands) and need more time to make all records done

**Comment by amarzavery** [comment]
Ok sounds good. I just inquired as I did not see them.

**Pull request merged**